### PR TITLE
Update search-the-audit-log-in-security-and-compliance.md

### PR DESCRIPTION
**Pull request by sujitnaray**

### CODE REVIEW

**Comment by sujitnaray** [comment]
@markjjo - FYI, there's another related change I'm making soon in the Turn logging ON/OFF page.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -61,10 +61,7 @@ Need to find if a user viewed a specific document or purged an item from their m
 
 Be sure to read the following items before you start searching the audit log.
 
-- You (or another admin) must first turn on audit logging before you can start searching the audit log. To turn it on, click **Turn on auditing** on the **Audit log search** page in the Security & Compliance Center. (If you don't see this link, auditing has already been turned on for your organization.) After you turn it on, a message is displayed that says the audit log is being prepared and that you can run a search in a couple of hours after the preparation is complete. You only have to do this once. For more information, see [Turn audit log search on or off](turn-audit-log-search-on-or-off.md).
-
-  > [!NOTE]
-  > We're in the process of turning on auditing by default. Until then, you can turn it on as previously described.
+- Audit Logging is turned on for enterprise customers, by default, but please ensure that this is the case. For more information, see [Turn audit log search on or off](turn-audit-log-search-on-or-off.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
- Audit logging is turned on for Enterprise customers, by default, but please ensure that this is the case. For more information, see [Turn audit log search on or off](turn-audit-log-search-on-or-off.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Comment by markjjo** [comment]
@sujitnaray PR merge. Should go live by EOD

**Pull request merged**